lang/block.go

```diff
@@ -109,24 +118,53 @@ func ParseBlocks(cfg *Config, blocks []comment.Block, inline bool) []*Block {
 }
 
 func printText(b *strings.Builder, text ...comment.Text) {
-	for i, t := range text {
-		if i > 0 {
-			b.WriteRune(' ')
-		}
-
+	for _, t := range text {
 		switch v := t.(type) {
 		case comment.Plain:
 			b.WriteString(string(v))
 		case comment.Italic:
 			b.WriteString(string(v))
 		case *comment.DocLink:
-			printText(b, v.Text...)
+			b.WriteString(printDocLink(v))
 		case *comment.Link:
-			printText(b, v.Text...)
+			b.WriteString(fmt.Sprintf("%s(%s)", v.Text, v.URL))
 		}
 	}
 }
 
+func printDocLink(docLink *comment.DocLink) string {
+	// Case [Volume]
```

Comment:
The comments give me an idea what you might mean but not necessarily anybody outside our organization. Did you mean sth. like this?
```
	// case: link a symbol within the same type, f. i. [Volume]
	// case: link a symbol within the same file or package [core.Volume]
        // case: link an externally symbol outside the same file or package [os.File]
```